src/Api/Auth/Models/Response/WebAuthn/WebAuthnCredentialResponseModel.cs

```diff
@@ -0,0 +1,20 @@
+﻿using Bit.Core.Auth.Entities;
+using Bit.Core.Models.Api;
+
+namespace Bit.Api.Auth.Models.Response.WebAuthn;
+
+public class WebAuthnCredentialResponseModel : ResponseModel
+{
+    private const string ResponseObj = "webauthnCredential";
+
+    public WebAuthnCredentialResponseModel(WebAuthnCredential credential) : base(ResponseObj)
+    {
+        Id = credential.Id.ToString();
+        Name = credential.Name;
+        PrfSupport = false;
+    }
+
+    public string Id { get; set; }
+    public string Name { get; set; }
+    public bool PrfSupport { get; set; }
```

Comment:
Technically this should be removed since this PR is not supposed to contain anything related to PRF. I'm probably going to be converting this into an `Enum` in the next PR making the name more accurate, can we just ignore this field for now?